src/Units/Unit.java

```diff
@@ -100,7 +100,11 @@ public boolean capture(MapMaster map)
     int captureThreshold = map.getEnvironment(x, y).terrainType.getCaptureThreshold();
     if( captureProgress >= captureThreshold )
     {
-      map.setOwner(CO, x, y);
+      Commander newOwner = CO;
+      // Capture territory for your primary CO in persistent tags
+      if( CO.gameRules.tagMode.supportsMultiCmdrSelect )
+        newOwner = CO.army.cos[0];
+      map.setOwner(newOwner, x, y);
```

Comment:
Why is unit setting the property owner? Isn't that handled by `CaptureEvent`?

Reply:
I'm pretty sure it just calls `unit.capture()`

Reply:
So it does. Though `CaptureEvent` also has this interesting snippet in its constructor:
```java
captureAmount = unit.getHP(); // TODO: Apply CO buffs.
```

Reply:
To be fair, Unit probably shouldn't own this game logic.
That said, `CaptureEvent()` looks like this on my end:
```java
  public CaptureEvent( Unit u, MapLocation loc )
  {
    unit = u;
    location = loc;
    XYCoord unitXY = new XYCoord(u.x, u.y);
    if( null != location && location.isCaptureable() && unit.CO.isEnemy(location.getOwner()) )
    {
      priorCaptureAmount = (unitXY.equals(location.getCoordinates()) ? unit.getCaptureProgress() : 0);
      UnitContext uc = new UnitContext(unit);
      captureAmount = uc.calculateCapturePower();
    }
    else
    {
      System.out.println("WARNING! Attempting to capture an invalid location!");
      priorCaptureAmount = 0;
      captureAmount = 0;
    }
  }
```
You may be looking at old code.

Reply:
So I am. Not sure how that happened.